### ISSUE TITLE
User is deleted from loft UI

### ISSUE THREAD

**Issue by bagulm123** [loft-sh/loft#186]
User is able to delete himself from the UI even if he is the first and only one user for loft UI.

There is no confirmation box on UI to let the user know about the side effects of deleting the user.

Steps to reproduce
1 loft start
2 loft login [https://localhost:9898](https://localhost:9898)  --insecure
3 Login to loft UI
4 Delete the user from UI, user is deleted. User lost access to UI now and not even able to reset the password. Below message is being observed.

> \~ ❯ loft reset password --user admin
> \[info\]   Resetting password of user admin
> \[fatal\]  user admin was not found, run with '--create' to create this user automatically

**Comment by LukasGentele** [comment]
@mahendrabagul Good catch! We should definitely prevent user self-deletion. I cannot think of any use case where this would be intended user behavior.
